src/cpp/processquit.cpp

```diff
@@ -102,6 +102,9 @@ inline bool is_gui_app()
 class ProcessQuit::Private : public QObject
 {
 	Q_OBJECT
+
+	Connection activatedConnection;
+
```

Comment:
I'd move this to the public section below. Typically, whenever we declare a "Private" class, all its vars are public.